package.json

```
@@ -6,7 +6,8 @@
   "types": "types/index.d.ts",
   "scripts": {
     "lint": "standard",
-    "test": "npm run test:unit && npm run test:typescript",
+    "lint:fix": "standard --fix",
+    "test": "npm run lint && npm run test:unit && npm run test:typescript",
```

The lint runs here: https://github.com/fastify/fastify-sensible/blob/2f867ba512fd0b1365b4a4bab36d5014af02786a/.github/workflows/ci.yml#L23

```suggestion
    "test": "npm run test:unit && npm run test:typescript",
```

---

Oh I see, I made that change because I didn't know if public repositories had unlimited runner minutes or not

Anyway reverting is a non-problem but just to let you know that many packages like [static](https://github.com/fastify/fastify-static/blob/e505f561a062183c1b79968e4e08ea98507ee379/package.json#L12) and fastify [itself](https://github.com/fastify/fastify/blob/ef7408a3feb377838e2aa580a4c05e34e00fdf1a/package.json#L22) have this check in their test script to combine with `pre-commit`

---

Thanks for pointing out

- static should be edited as well
- Fastify has its own ci, it does not run the `test` script:

https://github.com/fastify/fastify/blob/ef7408a3feb377838e2aa580a4c05e34e00fdf1a/.github/workflows/ci.yml#L137

---

Alright! So we remove lints from the test scripts and move linting to CIs?

Here's multipart (although it doesn't have lint: true in its CI): https://github.com/fastify/fastify-multipart/blob/80a01f41c03cc29a074742438b70d95261bb56ff/package.json#L48C26-L48C26